sqlglot/tokens.py

```diff
@@ -270,9 +270,17 @@ class _Tokenizer(type):
     def __new__(cls, clsname, bases, attrs):
         klass = super().__new__(cls, clsname, bases, attrs)
 
-        klass.QUOTES = [
-            prefix + quote for prefix in ["", "r"] for quote in klass.QUOTES
-        ]
+        klass.QUOTES = dict(
+            (quote[0], quote[1]) if isinstance(quote, tuple) else (quote, quote)
```

Comment:
let's switch all of these to be if isinstance(quote, str) inverse so that tuples, and lists could work

Reply:
Fixed